pages/api/sessions.ts

```diff
@@ -1,16 +1,11 @@
 // add api code here
 import { NextApiRequest, NextApiResponse } from 'next';
 import Stripe from 'stripe';
+import { stripeClient } from '../../utils/client';
 
-export type StripeSession = Stripe.Checkout.Session;
+export type StripeSession = Stripe.Checkout.Session & { url: string };
 export type EndpointResponse = { session: StripeSession } | { error: string };
 
-// 1. connect with stripe
-// auth with stripe
-const stripeClient = new Stripe(process.env.STRIPE_SECRET_KEY!, {
-  apiVersion: '2022-08-01',
-});
-
 export default async function handler(
```

Comment:
- probably `sessionsHandler` instead of `handler`
- a return type on this function would prevent mistakes from returning the incorrect data